tests/test_lhe_reader.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Filename: test_lhe_reader.py
+
+__author__ = "Johannes Schumann"
```

Comment:
I think we might need to remove these since the license etc is handled by LICENSE and it'd be difficult to track the authors this way (git blame should wook)

Reply:
Yeah no problem!